tests/lib/nm/wired_test.py

```diff
@@ -42,6 +42,13 @@ def test_create_setting_duplicate(NM_mock):
         base_profile.get_setting_wired.return_value.duplicate.return_value
 
 
+def test_create_setting_mac(NM_mock):
+    setting = nm.wired.create_setting(
+        {'mac-address': '01:23:45:67:89:ab'}, None)
```

Comment:
AFAIU, mac[0] & 0x02 should be true for local MAC addresses, i.e. use  `02:...` or `03:...` here.

Reply:
This is a unit test, who cares about these technicalities? It is also not wrong to use non local ones.
The usage or no usage of it matters for a production system.
Not even sure if it matters for an integration tests, although you may argue there that using a non-local one may collide with existing address.

Reply:
If we can use any address, we can also use a correct one. It is not a blocker for me, just an observation.

Reply:
Btw. it would be nice to also have integration tests, ideally for each interface type to ensure that it works across all types.

Reply:
The missing integration tests is a hard nack here.